includes/Modules/Idea_Hub.php

```diff
@@ -176,6 +177,18 @@ public function register() {
 			 * Show admin notices on the posts page if we have saved / new ideas.
 			 */
 			add_filter( 'googlesitekit_admin_notices', $this->get_method_proxy( 'admin_notice_idea_hub_ideas' ) );
+
+			/**
+			 * Adds a special class name to idea posts.
+			 */
+			add_filter( 'post_class', $this->get_method_proxy( 'update_post_classes' ), 10, 3 );
+
+			add_action(
+				'admin_footer',
+				function() {
+					echo '<div id="js-googlesitekit-post-list" class="googlesitekit-page"></div>';
```

Comment:
`Screen`s use the `googlesitekit-page` class on their render targets, but we shouldn't use that here since we're not rendering a screen. I'm not sure if there's anything actually going in here, but if there is, it should always be wrapped with a `googlesitekit-plugin` class.
```suggestion
					echo '<div id="js-googlesitekit-post-list" class="googlesitekit-plugin"></div>';
```

Reply:
Updated.